back-end/database/db_ops.py

```diff
@@ -1,35 +1,38 @@
 from typing import List, Tuple
```

Comment:
We should be using ORM objects everywhere instead of these SQL templates. I believe this file is no longer needed and we should probably remove it. Could you also double check?